MIT.md

```diff
@@ -1,10 +0,0 @@
-## Copyright 2021, [YOUR NAME]
```

Comment:
- [ ] [OPTIONAL] Please kindly fill your MIT license file including your name and change `[APP TYPE]` to web app. If I understood well  it should be in your dev branch